.github/workflows/maven.yml

```diff
@@ -1,25 +0,0 @@
-# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```

Comment:
I removed this file completely as Github would require access to Secmakrer's wsdl to be able to build the project.

Reply:
That's a shame, but yeah, looks impossible to solve that.

Reply:
Can we do it now that the wsdl's are committed to the repo?

Reply:
Definitely. I will bring back the workflow file.